types/nyaapi/index.d.ts

```diff
@@ -134,7 +175,7 @@ export namespace pantsu {
     interface SearchOptions {
         term?: string;
         n?: number;
-        c?: string[];
+        c?: Category;
```

Comment:
@infanf: You've replaced `string` with `Category` in a few places, but here you're replacing `string[]` -- is this intentional?

Reply:
@elibarzilay yes it is. The nyaa pantsu REST API docs where the array came from are outdated (or were wrong to begin with)